### PR TITLE
chore: update minimum versions

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: docs/minimum-versions.md [hunk 1]
```diff
@@ -50,10 +50,14 @@ https://help.keyman.com/developer/engine/android/latest-version/
 |-----------------------------------|--------------|
 | KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER | noble |
 | KEYMAN_MIN_TARGET_VERSION_ANDROID | 5 |
```
Comment (darcywong00) [inline docs/minimum-versions.md:52]:
```suggestion
| KEYMAN_MIN_TARGET_VERSION_ANDROID | 5.0 |
```

I should have made this clearer so not to confuse 5.1 (SDK 22)

Comment (darcywong00) [reply docs/minimum-versions.md:52]: nvm - will suggestion on the .sh

File: resources/build/minimum-versions.inc.sh [hunk 2]
```diff
@@ -15,7 +15,14 @@ KEYMAN_MIN_TARGET_VERSION_IOS=12.2            # iOS 12.2
 KEYMAN_MIN_TARGET_VERSION_WINDOWS=10          # Windows 10
 KEYMAN_MIN_TARGET_VERSION_MAC=10.13           # MacOS 10.13 (High Sierra)
 KEYMAN_MIN_TARGET_VERSION_UBUNTU=20.04        # Ubuntu 20.04 Focal
-KEYMAN_MIN_TARGET_VERSION_CHROME=95.0         # Final version that runs on Android 5.0
+KEYMAN_MIN_TARGET_VERSION_ANDROID_CHROME=53.0 # min vesrion of Chrome for Keyman for Android
```
Comment (darcywong00) [inline resources/build/minimum-versions.inc.sh:18]:
```suggestion
KEYMAN_MIN_TARGET_VERSION_ANDROID_CHROME=53.0 # min version of Chrome for Keyman for Android
```

File: docs/minimum-versions.md (hunk 1 above)
Comment (sgschantz) [reply docs/minimum-versions.md:52]: minimum versions are correct for mac and ios but I expect them to change before we release 18 as we make the move to Xcode 16

**Review by sgschantz — APPROVED** [review]

**Review by darcywong00 — APPROVED** [review]
lgtm for Android
minor nits on 5.0 and a typo in comment

**Review by ermshiperete — APPROVED** [review]

**Review by ermshiperete — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.138-alpha](https://keyman.com/downloads/releases/alpha/18.0.138)

**Pull request merged**